applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -477,14 +473,18 @@ static int change_active_brdcast_src(void)
 	if (++active_stream.brdcast_src_name_idx >= ARRAY_SIZE(brdcast_src_names)) {
 		active_stream.brdcast_src_name_idx = 0;
 	}
+	LOG_INF("Switching to %s", brdcast_src_names[active_stream.brdcast_src_name_idx]);
 
-	LOG_DBG("Switching to %s", brdcast_src_names[active_stream.brdcast_src_name_idx]);
-
+	LOG_DBG("Restarting scanning for broadcast sources");
 	ret = bt_audio_broadcast_sink_scan_start(BT_LE_SCAN_PASSIVE);
-	if (ret) {
+	if (ret && ret != -EALREADY) {
+		LOG_ERR("Unable to start scanning for broadcast sources");
 		return ret;
 	}
 
+	/* Always start playing new source */
+	playing_state = true;
```

Comment:
Do we need this line? Is this parameter true already?

Reply:
It could be in a playing_state=false at this point if you have stream, pause it and then switch to another broadcaster